CHANGELOG.md

```diff
@@ -9,13 +9,18 @@ All notable changes to this project will be documented in this file.
 - nifi: Add NiFi hadoop Azure and GCP libraries ([#943]).
 - base: Add containerdebug tool ([#928]).
 
+### Changed
+
+- all java products: These now use the Stackable Nexus build-repo by default instead of pulling from Maven central ([#953])
```

Comment:
```suggestion
- all java products: These now use the Stackable Nexus build-repo by default instead of pulling from Maven central ([#953]).
- all java products: Maven is now consistently run with `--batch-mode` and `--no-transfer-progress` to reduce noise ([#953]).
```